image_width: 752
image_height: 480
camera_name: narrow_stereo/left
camera_matrix:
  rows: 3
  cols: 3
  data: [462.40854,   0.     , 365.76766,
           0.     , 461.32479, 246.86042,
           0.     ,   0.     ,   1.     ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.283117, 0.072959, 0.000239, -0.000093, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [ 0.99999813, -0.00166958,  0.00097681,
          0.00166167,  0.9999663 ,  0.00803939,
         -0.0009902 , -0.00803775,  0.99996721]
projection_matrix:
  rows: 3
  cols: 4
  data: [440.71   ,   0.     , 371.06175,   0.     ,
           0.     , 440.71   , 254.30215,   0.     ,
           0.     ,   0.     ,   1.     ,   0.     ]
